build.sbt

```diff
@@ -89,27 +89,48 @@ lazy val scalacOptions212 = Seq(
   "-Ywarn-value-discard"               // Warn when non-Unit expression results are unused.
 )
 
+// scalac 2.13 flags, adapted from the 2.12 ones above, but with the flags that have been removed taken out. Next to
+// each item is a link to the Scala commit that explains why it was removed.
+lazy val scalacOptions213 = (scalacOptions212.toSet -- Set(
+  "-encoding", "utf-8", // No longer supported, UTF-8 is assumed.
+  "-Yno-adapted-args", // https://github.com/scala/scala/commit/a82a56ea5ba1ff0fb3b69c0963d4aec620ab68ad,
+  "-Ypartial-unification" // https://github.com/scala/scala/commit/7d5e0b01c7f645e4f727f704a18f93ce5c69a9dd
+)).toSeq
+
 def scalacOptionsVersion(scalaVersion: String) = {
   CrossVersion.partialVersion(scalaVersion) match {
+    case Some((2, scalaMajor)) if scalaMajor == 13 => scalacOptions213
     case Some((2, scalaMajor)) if scalaMajor == 12 => scalacOptions212
     case _ => scalacOptions211
   }
 }
 
 scalacOptions in ThisBuild ++= scalacOptionsVersion(scalaVersion.value)
 
+// Adds a `src/main/scala-2.13+` source directory for Scala 2.13 and newer
+// and a `src/main/scala-2.13-` source directory for Scala version older than 2.13
+val scala213Compat = Seq(
```

Comment:
This is directly taken from the suggested method by the Scala score team at https://github.com/scala/collection-strawman/wiki/FAQ#how-do-i-cross-build-my-project-against-scala-212-and-scala-213, so the naming is not my own (and is consistent with the naming used in all the big projects that have also used this method now, eg. Scalaz, Cats, Specs2),